src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DoNotRaiseExceptionsInUnexpectedLocations.cs

```diff
@@ -340,7 +340,7 @@ private static bool IsImplicitCastOperator(IMethodSymbol method, Compilation com
         {
             if (!method.IsStatic || !method.IsPublic())
                 return false;
```

Comment:
Might be worth enforcing curly braces.

Reply:
Brace enforcement is already configured. The analyzer was ported to the code style package in https://github.com/dotnet/roslyn/pull/42263, and is configured here:

https://github.com/dotnet/roslyn-analyzers/blob/1b6fbe0eafcdf4b9f795cbde00365d92e0dfd71f/.editorconfig#L118